subsys/nrf_security/src/drivers/cracen/common/include/cracen/math_helpers.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+/**
+ * @brief Decode a byte array containing a coordinate in place.
+ *
+ * @note Corresponds to the decodeUCoordinate function in RFC 7748.
+ *
+ * @param[in] u    Byte array containing a coordinate.
+ * @param[in] size Size of the coordinate to be decoded in bytes.
+ */
+void decode_u_coordinate(uint8_t *u, uint8_t size);
```

Comment:
the implementation here only does decodeUCoordinate for 25519. For 448 decodeUCoordinate does not nothing. Maybe rename the function to ...25519 and remove the size parameter, or update the implementation to match the RFC? I see the RFC takes the size in bits.